index.html

```diff
@@ -5,26 +5,28 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Portfolio</title>
   <script src="https://kit.fontawesome.com/39e3f91239.js" crossorigin="anonymous"></script>
-  <link rel="stylesheet" href="desktopVersion.css">
-  <link rel="stylesheet" href="small_version.css">
+  <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
     <header>
-    <nav>
-      <a class="logo"href="#homa">Emmabgra</a>
-      <img class="menu" src="img/Union.svg" alt="">
-      <ul class="navbar__ul show__links">
-        <li class="navrbar__link"><a href="#portfolio">Portfolio</a></li>
-        <li class="navrbar__link"><a href="#about">About</a></li>
-        <li class="navrbar__link"><a href="#contact">Contact</a></li>
+      <nav class="nav">
+        <a class="logo"href="#homa">Emmabgra</a>
+        <img class="menu" src="img/Union.svg" alt="">
+       <ul class="navbar__ul" id="navbar__ul">
+        <i class="fa-solid fa-xmark"></i>
+        <li><a href="#contact" class="navMenu">Contact</a></li>
+        <li><a href="#portfolio" class="navMenu">Portfolio</a></li>
+        <li><a href="#about" class="navMenu">About</a></li>
       </ul>
-    </nav>
-    <div class="togg">
-      
-    </div>
-  </header>
+     </nav>
+    </header>
+
+
```

Comment:
- You have done well so far, but there are some differences between your mobile menu design and the design in the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- Please note that the background is blurred when the mobile menu is displayed, please fix this. I suggest you use the `back-drop filter` CSS property to fix this 🙏 
- Also, please make sure that the links in the mobile menu are arranged the same way as the Figma template 🙏 
![hrtrtr](https://github.com/sibomana123/MV__Portfolio/assets/73607512/edf57a0d-fcd4-4fe6-a5f2-77f9d6c6b5ff)

Figma template:
![mfmchm](https://github.com/sibomana123/MV__Portfolio/assets/73607512/948e1c5f-efbd-43bb-989b-62d9c483e47d)